src/unicode.hh

```diff
@@ -20,12 +20,42 @@ inline bool is_eol(Codepoint c) noexcept
 
 inline bool is_horizontal_blank(Codepoint c) noexcept
 {
-    return c == ' ' or c == '\t';
+    // Characters considered whitespace by ECMA Regex Spec
+    // <https://262.ecma-international.org/11.0/#sec-white-space>
+    return c == '\t'      or
+           c == '\v'      or
```

Comment:
A vertical tab is not a horizontal blank, see

`printf '\v' | perl -ne 'print if m/\h/'`

Reply:
Fair enough, fixed.
Test added.
I looked through the rest of the characters, but I did not see any other vertical space characters that are also not line terminators.